Cargo.toml

```diff
@@ -36,11 +36,15 @@ tempfile = "3.8"
 arrow = { version = "50.0", default-features = false, features = ["ffi"] }
 
 [workspace]
-members = ["gdal-sys"]
+members = ["gdal-src", "gdal-sys"]
 
 # docs.rs-specific configuration
 [package.metadata.docs.rs]
 # include `array` feature in documentation
 features = ["array"]
 # define attribute `docsrs` for feature badges
 rustdoc-args = ["--cfg", "docsrs"]
+
+
+[patch.crates-io]
+proj-sys = { git = "https://github.com/GiGainfosystems/proj", rev = "44a503869709ea793a5f8452bfb9e2a99b7160fc" }
```

Comment:
Is this temporary until your three `proj` PRs get merged? Do you want to merge this before them?

Reply:
> Is this temporary until your three proj PRs get merged? Do you want to merge this before them?

It only depends on https://github.com/georust/proj/pull/190. The other two PR's are not needed for this feature, although I obviously would prefer getting all of them merged.

> Definitely don't think this should be merged like this. Need to be pointing to the official version.

This is unfortunately still waiting on a review from the relevant maintainers of the georust project.

> Why are we interacting directly with proj-sys anyway? Should we not be going through the C++ gdal wrappers around this? 

There are two reasons for depending on `proj-sys`:

* Statically linking gdal does not link libproj. By depending on `proj-sys` we add a dependency on libproj that is resolved by rustc while linking the final binary.
* gdal requires access to the proj header files for building. By enabling the bundled feature of `proj-sys` we can access the relevant include directory there to get access to the header files

> What happens if the user links in georust/proj into their application? Do we end up with multiple copies of proj?

As it's a `-sys` crate, cargo ensures that there will be only one version of this crate in your dependency tree. Also keep in mind that this all is only relevant if you enable the `bundled` feature for gdal and also depend on `proj-sys`.